doc/znapzend.pod

```diff
@@ -1,4 +1,4 @@
-=head1 NAME
+[01;31m[K=head1 NAME[m[K
```

Comment:
```suggestion
=head1 NAME
```

Reply:
Hmm, different tooling is differently eager it seems :)